pkgs/tools/misc/codebraid/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "codebraid";
+  version = "0.5.0-unstable-2019-12-11";
+
+  src = fetchFromGitHub {
+    owner = "gpoore";
+    repo = pname;
+    rev = "fac1b29";
+    sha256 = "0ldfrkkip7i1fdyz1iydyik3mhm0xv0jvxnl37r7g707dl38vf3h";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ bespon ];
+
```

Comment:
```suggestion
  # unfortunately upstream doesn't contain tests
  checkPhase = ''
    $out/bin/codebraid --help > /dev/null
  '';
```

Reply:
codebraid: add trivial check in lieu of unit tests